src/models/Square.ts

```diff
@@ -1,6 +1,11 @@
 class Square {
-    x!: number;
-    y!: number;
+    row: number;
+    column: number;
+
+    constructor(row: number, column: number) {
+        this.row = row;
+        this.column = column;
```

Comment:
tu taki trochę przerost formy nad treścią? ja jak gdzieś pisałam jestem raczej za modelami w formie interfejsów, ale jeżeli już klasa to czemu odeszłaś od pierwotnego zapisu?

Reply:
?

Reply:
aktualnie na repo jest w formie klasy. Myślę ze później można zrobić refaktoryzację

Reply:
chodziło mi tylko o formę zapisu, tu dla przykładu: https://www.stevefenton.co.uk/2013/04/stop-manually-assigning-typescript-constructor-parameters/
Marcin użył jeszcze bardziej skróconej formy
ale tak jak piszesz, imo to będzie do refaktoru, na razie daję okejkę

Reply:
done